src/util.js

```diff
@@ -241,7 +241,7 @@ export function zdiv(n, d) {
  * @param {number} hi - used to specify a the high end of a subset of the list
  * @returns number
  */
-export function bisectLeft(arr, value, lo=0, hi=arr.length) {
+export function bisectLeft (arr, value, lo=0, hi=arr.length) {
```

Comment:
Non-blocking but we may want to add a rule that the `=` here needs spaces around it.

Reply:
@LeaVerou I added a few more spacing-related rules in https://github.com/color-js/color.js/pull/373/commits/0e14ea1dd8cc7c896790a27b61b6f5322b1b3736 -- let me know if you want me to remove any of them.